src/common-components/ThirdPartyAuth.jsx

```diff
@@ -20,20 +24,32 @@ import {
 const ThirdPartyAuth = (props) => {
   const { formatMessage } = useIntl();
   const {
-    providers, secondaryProviders, currentProvider, handleInstitutionLogin, thirdPartyAuthApiStatus,
+    providers, 
+    secondaryProviders, 
+    currentProvider, 
+    handleInstitutionLogin, 
+    thirdPartyAuthApiStatus, 
+    helpingText, 
+    isLogin,
```

Comment:
can we please rename it, so that it's more meaningful?
```suggestion
    isLoginPage,
```

Reply:
addressed